src/main/resources/org/csanchez/jenkins/plugins/kubernetes/KubernetesCloud/help-directConnection.html

```diff
@@ -0,0 +1,6 @@
+<div>
+  When this option is enabled a <a href="https://github.com/jenkinsci/remoting/blob/master/docs/tcpAgent.md#connect-directly-to-tcp-port">direct connection</a>
```

Comment:
Use `target="_blank"`, and never link to a GitHub branch head reference from the product, as the link could break at any moment. You could use a tagged version, but better is to create a redirect on jenkins.io (example: https://github.com/jenkins-infra/jenkins.io/pull/2290), or simply make sure that everything the user needs to know is listed here.

Reply:
done